ui/src/app/applications/components/application-details/application-resource-list.tsx

```diff
@@ -31,35 +31,43 @@ export const ApplicationResourceList = ({
         return null;
     }
     const parentNode = ((resources || []).length > 0 && (getResNode(tree.nodes, nodeKey(resources[0])) as ResourceNode)?.parentRefs?.[0]) || ({} as ResourceRef);
+    const searchParams = new URLSearchParams(window.location.search);
+    const view = searchParams.get('view');
 
+    const ParenRefDetails = () => {
+        return (
+            <div className='resource-parent-node-info-title'>
+                <div>Parent Node Info</div>
+                <div className='resource-parent-node-info-title__label'>
+                    <div>Name:</div>
+                    <div>{parentNode?.name}</div>
+                </div>
+                <div className='resource-parent-node-info-title__label'>
+                    <div>Kind:</div>
+                    <div>{parentNode?.kind}</div>
+                </div>
+            </div>
+        );
+    };
     return (
         <div>
-            <div className='resource-details__header' style={{paddingTop: '20px'}}>
-                {Object.keys(parentNode).length > 0 && (
```

Comment:
Did you want to show this info/text when there is no parent node info? eg.:

```
        return (Object.keys(parentNode).length > 0 ? (
            <div className='resource-parent-node-info-title'>
                <div>Parent Node Info</div>
                <div className='resource-parent-node-info-title__label'>
                    <div>Name:</div>
                    <div>{parentNode?.name}</div>
                </div>
                <div className='resource-parent-node-info-title__label'>
                    <div>Kind:</div>
                    <div>{parentNode?.kind}</div>
                </div>
            </div> ) : (
                <div></div>
            )
        );
```

Reply:
>Did you want to show this info/text when there is no parent node info? eg.:

Added the code, if no parent information is available, do not display anything. Additionally, I would like the parent information to appear only when viewing the resource list (pod/replicaset) in tree view or network view and the total pod count exceeds 15.